### ISSUE TITLE
What is the ground truth for training the Pose Refinement Module?

### ISSUE THREAD

**Issue by hahv** [sj-li/PR-GCN#1]
Thank you for sharing the code.

As described in the paper (and Readme also), I know that the pose/skeleton data  extracted from OpenPose (Kinetics dataset), and Kinect v2 SDK (for NTU RGB+D). 

My question is:

1. Since the pose data is automatically extracted from software, it can contain errors. Do you have the groundtruth for pose data?
2. Do we need groundtruth for pose data? If there is no ground truth, it means the pose refiment module 's weights are learned by backprop from the label of action, doesn't it?

Thank you for your clarification.
